builds/jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -451,10 +453,16 @@ def handleDynamoDbStreamError(deployOutput, service_config, environment_logical_
   }
 }
 
-def deleteAndRedeployService(environment_logical_id, envBucketKey){
+def deleteAndRedeployService(environment_logical_id, credsId, env_key, accountDetails){
   echoServerlessFile()
-  sh "serverless remove --stage $environment_logical_id -v --bucket ${configLoader.AWS.S3[envBucketKey]}"
-  def redeployOutput = servelessDeploy(environment_logical_id, envBucketKey)
+  def s3BucketNameValue;
```

Comment:
Same code line number 458 to 463 is repeating from line number 478. Please use a method instead